asteroidsyncserviced/asteroidsyncserviced.pro

```diff
@@ -3,8 +3,9 @@ QT -= gui
 
 WATCHFISH_FEATURES = notificationmonitor walltime music
 
+SUBDIRS += libwatchfish
```

Comment:
Shouldn't this be put inside starfish.pri ?

Reply:
done

Reply:
What about "SUBDIRS += libwatchfish" ?

Reply:
If i only include "libwatchfish.pri" it is not possible for me to build the projekt.
In the .pro file we have "QT += dbus" and without that i can not compile it.

Reply:
But what about putting the "SUBDIRS += libwatchfish" line inside starfish.pri ?

Reply:
Is libwatch only needed by starfish? If yes, than i put it inside starfish.pri. For sure...

Reply:
Afaik libwatchfish is mostly sailfishos-specific and wouldn't work on UBPorts.

Reply:
Thank you so much for the explanation. I have not yet understood and looked at all this so deeply.